cmd/loop/liquidity.go

```diff
@@ -445,11 +459,37 @@ func setParams(ctx *cli.Context) error {
 		flagSet = true
 	}
 
-	if ctx.IsSet("destaddr") {
+	switch {
+	case ctx.IsSet("destaddr") && ctx.IsSet("account"):
+		return fmt.Errorf("cannot set destaddr and account at the " +
+			"same time")
+
+	case ctx.IsSet("destaddr"):
 		params.AutoloopDestAddress = ctx.String("destaddr")
+		params.Account = ""
+		flagSet = true
+
+	case ctx.IsSet("account"):
+		params.Account = ctx.String("account")
+		params.AutoloopDestAddress = ""
 		flagSet = true
 	}
 
+	if ctx.IsSet("account") != ctx.IsSet("account_addr_type") {
```

Comment:
nit: validation could be before the params switch.

Reply:
I think I could also make it part of the switch itself.